src/main/webapp/resources/js/pages/analysis/components/AnalysisDelete.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useContext } from "react";
+import { Button, Checkbox, Alert, Popconfirm, Row } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { showNotification } from "../../../modules/notifications";
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+import { deleteAnalysis } from "../../../apis/analysis/analysis";
+
+export default function AnalysisDelete() {
+  const { analysisContext } = useContext(AnalysisContext);
+  const [deleteConfirm, setDeleteConfirm] = useState(false);
+
+  function onChange(e) {
+    // Set local state for deleteConfirm
+    if (e.target.checked == true) {
+      setDeleteConfirm(true);
+    } else {
+      setDeleteConfirm(false);
+    }
+  }
+
+  function handleDeleteConfirm() {
+    if (deleteConfirm) {
+      deleteAnalysis(analysisContext.analysis.identifier).then(res =>
+        showNotification({ text: res.result })
+      );
+
+      window.setTimeout(function() {
+        window.location.replace(window.TL.BASE_URL);
+      }, 3500);
+    }
+  }
+
+  return (
+    <>
+      <h2 style={{ fontWeight: "bold" }}>
+        {getI18N("analysis.tab.delete-analysis")}
+      </h2>
+      <strong className="spaced-top__sm">
+        <Alert
+          message={getI18N(
+            "analysis.tab.content.delete.permanent-action-warning"
+          )}
+          type="warning"
+        />
+      </strong>
+      <Row className="spaced-top__lg">
+        <Checkbox onChange={onChange}>
+          {getI18N("analysis.tab.content.delete.checkbox-confirmation-label")}
+        </Checkbox>
+      </Row>
+      <Row>
+        { deleteConfirm ?
```

Comment:
Use `deleteConfirm` as the `disabled` attribute of the `PopConfirm` and button instead of rendering the button 2 different ways.  Ask me if this does not make sense.

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a